backend/services/wallet/src/routes.py

```diff
@@ -0,0 +1,61 @@
+from src import app
+from flask import request, jsonify
+import requests
+import sys, os
+import json
+
+if os.environ.get('SERVICE_IN_DOCKER',False):             
+    sys.path.append(os.path.abspath(os.path.join('..', '')))
+else:
+    sys.path.append(os.path.abspath(os.path.join('../..', '')))
+
+from shared import HttpCode, FailureReturnString
+
+@app.route("/")
```

Comment:
I think we could just remove this endpoint and its corresponding test, we don't really need it

Reply:
Do you mind if we keep these home endpoints until we have everything ready to ship? They are quite useful to check if the service is up while in dev.

Reply:
Yep fair enough, sounds good!